src/Tests/Microsoft.NET.Build.Tests/GlobalPropertyFlowTests.cs

```diff
@@ -86,7 +86,7 @@ public void TestGlobalPropertyFlowToLibrary(bool passSelfContained, bool passRun
 
             bool buildingSelfContained = passSelfContained || passRuntimeIdentifier;
 
-            ValidateProperties(testAsset, _testProject, expectSelfContained: buildingSelfContained, expectRuntimeIdentifier: buildingSelfContained);
+            ValidateProperties(testAsset, _testProject, expectSelfContained: buildingSelfContained, expectRuntimeIdentifier: passRuntimeIdentifier);
```

Comment:
I'm not sure this change is right.  If we pass `--self-contained` as a command-line option, then the RuntimeIdentifier should be inferred.

Reply:
You are right; It was very much not right, reverted this.